### DISCUSSION TITLE
Enquire about course details

### DISCUSSION THREAD

**Discussion by Sehgal-Shreyansh** [SartHak-0-Sach/Harkirat-Singh-course_code_and_notes#8] [General]
Hi Sarthak bhaiya, first of all love the efforts you have been doing to provide us with these neat and ellaborative code and notes free of cost here on github!

Secondly, I wanted to ask about the course contents, like what all can we see coming up in the future, how long is this course going to be, and can you share more resources for beginners who struggle to understand the fundamentals and can find this course content overwhelming.

Thanks,
Shreyansh

**Comment by Sehgal-Shreyansh** [comment]
I also forgot to add another question, about the approach we should take up while learning such concepts that can be difficult to understand in the first time like async await promises and concepts like Middlewares using express that you covered in week-3

**Reply by SartHak-0-Sach** [reply]
To learn difficult concepts like these, you need to have a positive mindset and connect them to the basics while learning them.
The only way to truly understand them is to give them the time they deserve and don't be lazy with them. Write your notes about any piece of info you find about them and try to actually implement it.

Nowadays you can also take help of tools like chatGPT for better visualization and understanding of concepts!!

**Comment by SartHak-0-Sach** [comment]
> Hi Sarthak bhaiya, first of all love the efforts you have been doing to provide us with these neat and ellaborative code and notes free of cost here on github!
> 
> Secondly, I wanted to ask about the course contents, like what all can we see coming up in the future, how long is this course going to be, and can you share more resources for beginners who struggle to understand the fundamentals and can find this course content overwhelming.
> 
> Thanks,
> Shreyansh

Hi Shreyansh, thank you so much for the lovely message.
It means a lot to me that you appreciated my efforts with such kind words😊

About the course content, I will for sure put the syllabus of 0-1 and 1-100 courses which will help you get all info about upcoming lectures.

Apart from that,
https://docs.google.com/document/d/1Atkd-R1Rvr1fcBY8NHCWRKlhSIfhPrrFSgXhkfIU3-0/edit

Here is a roadmap that I prepared for my students to learn basics of web development using a project based approach and covers all the fundamentals along with assignments to reach perfection in the same!!
